samples/cpp/tutorial_code/objectDetection/objectDetection.cpp

```diff
@@ -44,7 +55,7 @@ int main( void )
         //-- 3. Apply the classifier to the frame
         detectAndDisplay( frame );
 
-        int c = waitKey(10);
+        int c = waitKey(1);
```

Comment:
such a small delay may cause the program to process the same frame many times, because cameras do not usually capture video at 1000fps and we are not able to process it in 1ms